### DISCUSSION TITLE
CSV files with results

### DISCUSSION THREAD

**Discussion by nlemusd** [bhasin-lab/SurvivalGenie#4] [Ideas]
Dear Bhasin

Great Collection and pretty straightforward to use.

It will help many people, I am impressed by it.

Is there a chance to download the processed data as CSV or tab, from the figures generated by the SurvivalGenie?

It will be extremely helpful for the user 

Best

Nico

**Comment by bhasin-lab** [comment]
Yes Version 2 will have option to download CSV files

**Reply by nlemusd** [reply]
Great, Do you know approximately when version two will be release? 

This is fantastic thank you very much for all your effort
